### PR TITLE
Completed Todo List built with JS

### PR DESCRIPTION
**Pull request by RyelBanfield**
This PR contained the completed JavaScript Todo List project.

Users can create projects and add to-do items to each project. They can clear completed tasks and delete entire projects.

### CODE REVIEW

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Ryel Banfield 

THIS IS VERY IMPRESSIVE WORK. YOU HAVE DONE A GOOD JOB :+1:
However, there are some changes that you need to make in order to get your project approved

- [x] Make the develop branch the default branch

- [x] when I clear the task and click on the project name, the task again appears. Kindly remove the task from the local storage after clearing

- [ ] User can edit the details of a to-do item

- [ ] Kindly add ```"dev": "webpack --mode development",``` to your scripts in package.json file

- [ ] Project uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/))

- [ ] use arrow functions instead of the function keyword


## OPTIONAL

- [ ] It'd be great to follow a design from Behance for this project
[example](https://www.behance.net/gallery/107935847/Todo-List-Desktop-Mobile-app-UI-Design?tracking_source=search_projects_recommended%7Cto%20do%20list)

- [ ]  It'd be great if the forms used to create a project or a todo should have validations, you should not be able to create an empty project/todo

Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: dist/index.html [hunk 1]
```diff
@@ -1,14 +1,85 @@
...
+      <button class="add-todo w-100 btn btn-danger m-1">
+        <i class="bi bi-plus"></i>
+      </button>
+      <div class="todo-items mt-3">
+        <ul class="todo-items-list fs-4"></ul>
+      </div>
+      <div class="buttons d-flex justify-content-evenly my-3">
+        <!-- <button class="btn btn-danger m-1">Clear Tasks</button> -->
+        <!-- <button class="btn btn-danger m-1">Delete List</button> -->
+      </div>
```
Comment (MkrtichSargsyan) [inline dist/index.html:48]: Kindly remove the commented buttons.

File: src/index.js [hunk 2]
```diff
@@ -1 +1,122 @@
 import './style.css';
+// import { format } from 'date-fns';
```
Comment (MkrtichSargsyan) [inline src/index.js:2]: Remove the commented import statement

**Comment by RyelBanfield** [comment]
@MkrtichSargsyan `when I clear the task and click on the project name, the task again appears. Kindly remove the task from the local storage after clearing` the clear task button only clears tasks that you have marked completed or strikes through. The delete button deletes the project.

**Review by mke2111 — APPROVED** [review]
# 😄 🥇  STATUS: APPROVED  🥇 👏🏽 

## Hey @RyelBanfield, 👋🏽 
Congratulations, 🥂 . You got minimum requirements for approval. 👍🏽 

### **_Well Done_** 💯 

    1. Thanks for the implemented changes that were previously requested.
    2. The to-do app runs as expected
    

### **_Room for Improvement_** 🤪 

    1. The priority field should be validated. It is currently accepting number 9897
    2. Add a default project for your app.

### You may now merge to the develop branch and remember to submit a project completion form.
### I wish you the very best in your upcoming projects.

**Cheers 🍹 and Happy coding 💻** 

<hr>

### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)

**Pull request merged**